### PR TITLE
Pass MFA instead of fun

### PR DESCRIPTION
**Pull request by rnewson**
BugzID: 49513

### CODE REVIEW

File: src/fabric_doc_attachments.erl [hunk 1]
```diff
@@ -20,34 +20,36 @@
...
-                iolist_to_binary(Data)
-        after Timeout ->
-            exit(timeout)
-        end
-    end;
+    {?MODULE, write_stream, Middleman};
 receiver(_Req, Length) ->
     exit({length_not_integer, Length}).
 
+write_stream(Middleman) ->
```
Comment (davisp) [inline src/fabric_doc_attachments.erl]: This shoudl be write_stream(Middleman, ChunkFun), no?

Comment (davisp) [inline src/fabric_doc_attachments.erl]: nm. am dumb.

**Comment by davisp** [comment]
Same as the other, we'll want to provide an actual reason for this in the commit message.

Other than that, +1

File: src/fabric_doc_attachments.erl (hunk 1 above)
Comment (rnewson) [inline src/fabric_doc_attachments.erl]: :)

**Comment by rnewson** [comment]
will tidy in a bit

**Pull request merged**